src/lib/input-helpers.js

```diff
@@ -35,3 +35,13 @@ export const getSettingsErrors = inputs => {
 
   return errors;
 };
+
+export const getSaveGraphErrors = name => {
+  const errors = {};
+  if (!name.length) {
+    errors.name = 'Name required';
+  } else if (name.length >= 30) {
```

Comment:
Is there a reason this limit is so low? If we have a limit at all (which isn't unreasonable), I'd like to see it around 255 characters.

Reply:
Done